docs/backup-restore.md

```diff
@@ -0,0 +1,62 @@
+
+## Create Backup
+
+### For Transactional Databases
+
+#### JDBC databases
+
+You can take a backup with your favorite way for JDBC databases.
+One requirement for backup in Scalar DL on JDBC databases is that backups for all the Scalar DL managed tables (including the coordinator and scalardb tables) need to be transactionally-consistent or automatically recoverable to a transactionally-consistent state.
+That means that you need to create a consistent snapshot by dumping all tables in a single transaction. For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
+Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
+
+### For Non-transactional Databases
+
+#### Basic strategy to create a transactionally-consistent backup
+
+One way to create a transactionally-consistent backup is to take a backup while Scalar DL cluster does not have outstanding transactions.
+If an underlying database supports a point-in-time snapshot/backup mechanism, you can take a snapshot during the period.
+If an underlying database supports a point-in-time restore/recovery mechanism, you can set a restore point to a specific time (preferably the mid-time) in the period since the system takes backups for each operation in such a case.
+
+To easily achieve this, Scalar DL exposes pause API to make Scalar DL drain outstanding transactions and stop accepting new transactions.
+We also provide a simple client program called scalar-admin to make a pause request (and unpause request) to a Scalar DL cluster and obtain a paused duration.
+
+Note that when you use a point-in-time-restore/recovery mechanism, it is recommended to minimize the clock drifts between nodes (Scalar DL nodes and a client node that requests a pause) by using clock synchronization such as NTP.
+Otherwise, the time you get as a paused duration might be too different from the time in which the pause was actually conducted, which could restore to a point where ongoing transactions exist.
+Also, it is recommended to pause a long enough time (e.g., 10 seconds) and use the mid-time of the paused duration since clock synchronization cannot perfectly synchronize clocks between nodes.
+
+#### Database-specific ways to create a transactionally-consistent backup
+
+**Cassandra**
+
+Cassandra has a built-in replication mechanism, so you do not always have to create a transactionally-consistent backup.
+
+For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, you do not need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism. 
```

Comment:
```suggestion
For example, if replication is set to 3 and only the data of one of the nodes in a cluster is lost, you do not need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism. 
```

Reply:
Thank you for the suggestion.
Updated document with suggested change.